backend/src/main/java/at/ac/tuwien/sepm/groupphase/backend/endpoint/dto/AttemptInputDto.java

```diff
@@ -9,6 +9,8 @@
     private Long cardId;
     @NotNull
     private Status status;
+    @NotNull
+    private Boolean reverse;
```

Comment:
I think if we use `boolean` instead of `Boolean` we can spare us the `@NotNull`.